.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,49 @@ Thanks for contributing to this project!
 - Run `pnpm install` and `pnpm test`
 - In short: help us help you to get this through!
```

Comment:
Do we still want to keep first few lines in?

Reply:
Probably not. I'll remove it